mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -140,22 +156,27 @@ static Value makeBroadcast(PatternRewriter &b, MemRefType outType, Value inp,
 
       inpType = inp.getType().template cast<MemRefType>();
       inpShape = inpType.getShape();
+    } else if (inpIdxMap.isMinorIdentityWithBroadcasting(&bcastDims)) {
```

Comment:
Perhaps put a comment in the empty condition to indicate that it's intentionally empty.